Sources/MapboxCoreNavigation/EHorizon/RoadClass.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+import MapboxNavigationNative
+
+/**
+ Functional road class
+ See for details: https://wiki.openstreetmap.org/wiki/Key:highway
+ */
+public enum RoadClass {
```

Comment:
We should reuse `MapboxDirections.MapboxStreetsRoadClass` instead of defining a new enumeration.